org.hl7.fhir.publisher.core/src/main/java/org/hl7/fhir/igtools/publisher/SpecMapManager.java

```diff
@@ -183,7 +183,11 @@ public String getPath(String url, String def, String rt, String id) throws FHIRE
     }
     if (special != null) {
       switch (special) {
-      case Simplifier: return "https://simplifier.net/resolve?scope="+pi.name()+"@"+pi.version()+"&canonical="+url;
```

Comment:
I also still like the idea to see if Simplifier _can_ acually resolve the said query, and if not fall back to the plain `url`.

Perhaps this can be tested as part of the line 428 - when deciding whether to assign special=Simplifier, or fall back to special=null.

Reply:
note to self - this is primarily for urls from _dependent_ packages, not the current IG package.